datacake-crdt/src/timestamp.rs

```diff
@@ -51,7 +51,7 @@ pub const DATACAKE_EPOCH: Duration = Duration::from_secs(1672534861);
 /// let timestamp = node_b.send().unwrap();
 ///
 /// // Node-a gets this payload with the timestamp and so we call `recv()` on our clock.
-/// // This was node-a is also unique and monotonic.
+/// // This makes node-a is also unique and monotonic.
```

Comment:
```suggestion
/// // This makes node-a update its internal timestamp and counter ensuring subsequent events
/// // are always treated as coming after our timestamp, maintaining the causality of events.
```
I think this might have been one sentence originally, and then forgotten to be finished.